scripts/generate-llms-files.js

```diff
@@ -0,0 +1,120 @@
+  let files = [];
+  for (const item of items) {
+    if (item.link) {
+      files.push(item.link);
+    }
+    if (item.items) {
+      files = files.concat(getMarkdownFiles(item.items));
+    }
+  }
+  return files;
+}
+
+function generateLlmsTxt() {
+  let content = '# Api3 Docs\n\n';
+  content += `> Api3 Docs helps developers build dApps using Api3 data feeds and searchers recapture oracle extractable value (OEV).\n\n`;
+
+  const sidebars = {
+    dapps: dappsSidebar,
+    'oev-searchers': oevSearchersSidebar,
+  };
+
+  for (const key in sidebars) {
+    const sidebar = sidebars[key];
+    if (sidebar) {
+      content += `## ${key}\n\n`;
+      const files = getMarkdownFiles(sidebar);
+      for (const file of files) {
+        const filePath = path.join(docsDir, `${file}.md`);
+        if (fs.existsSync(filePath)) {
+          const fileContent = fs.readFileSync(filePath, 'utf-8');
+          const lines = fileContent.split('\n');
+          let title = path.basename(file, '.md');
+          for (const line of lines) {
+            if (line.startsWith('title: ')) {
+              title = line.substring('title: '.length);
+              break;
+            }
+          }
+          const url = `https://docs.api3.org${file}`;
+          content += `- [${title}](${url.replace(/\/$/, '/index')}.md)\n`;
+        } else {
+          const indexPath = path.join(docsDir, file, 'index.md');
+          if (fs.existsSync(indexPath)) {
+            const fileContent = fs.readFileSync(indexPath, 'utf-8');
+            const lines = fileContent.split('\n');
+            let title = path.basename(file);
+            for (const line of lines) {
+              if (line.startsWith('title: ')) {
+                title = line.substring('title: '.length);
+                break;
+              }
+            }
+            const url = `https://docs.api3.org${file}`;
+            content += `- [${title}](${url.replace(/\/$/, '/index.md')})\n`;
+          }
+        }
+      }
+      content += '\n';
+    }
+  }
+
+  fs.writeFileSync(llmsTxtPath, content);
+  console.log(`Successfully created ${llmsTxtPath}`);
+}
+
+function generateLlmsFullTxt() {
+  const llmsTxtContent = fs.readFileSync(llmsTxtPath, 'utf-8');
+  const links = llmsTxtContent.match(/- \[(.*?)\]\((.*?)\)/g);
+  let fullContent = '';
+
+  if (links) {
+    for (const link of links) {
+      const match = link.match(/- \[(.*?)\]\((.*?)\)/);
+      if (match) {
+        const url = match[2].replace('https://docs.api3.org', '');
+        const filePath = path.join(docsDir, url);
+        if (fs.existsSync(filePath)) {
+          let fileContent = fs.readFileSync(filePath, 'utf-8');
+          const pageHeaderIndex = fileContent.indexOf('<PageHeader/>\n\n');
+          if (pageHeaderIndex === -1) {
+            throw new Error(`Could not find PageHeader in ${filePath}`);
+          }
+          fileContent = fileContent.substring(
+            pageHeaderIndex + '<PageHeader/>\n\n'.length
```

Comment:
`<PageHeader/>\n\n` could be a variable since it's used twice, but not a big deal

Reply:
Done in https://github.com/api3dao/api3-docs/pull/222/commits/7a91fe2485688f2d3af6ff0b9f6f5fa51cb18fdf